dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScopeTest.kt

```diff
@@ -355,6 +358,26 @@ internal class RumViewManagerScopeTest {
         assertThat(testedScope.childrenScopes).hasSize(0)
     }
 
+    @Test
+    fun `𝕄 not start a bg ViewScope 𝕎 handleEvent { app displayed, event is background ANR}`() {
+        // Given
+        testedScope.applicationDisplayed = true
+        val fakeEvent = RumRawEvent.AddError(
+            message = ANRDetectorRunnable.ANR_MESSAGE,
```

Comment:
Probably off topic - from names `ANRDetectorRunnable.ANR_MESSAGE` and `ANRException` I can not drive understanding it is a background ANR.

Reply:
Also, testedScope has `applicationDisplayed` as `true`, which sounds to me as if it is not in the background.